scripts/cplane/kube_dashboard.sh

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env bash
+
+# Strip off errant 'localhost-y' reference that get created via vagrant
+CPLANE_IP=$(echo $(hostname -i | sed -E 's/127\.0\.[0-9]+\.[0-9]+//g'))
+
+# Kubernetes Dashboard Parameters
+NAMESPACE="kubernetes-dashboard"
+SERVICE="kubernetes-dashboard"
+DASHBOARD_PORT=32443
+
+function help_msg {
+    echo ""
+    echo "usage: ${0} [worker|cplane|token], where:"
+    echo "    worker - Deploy dashboard on any worker node"
+    echo "    cplane - Deploy dashboard on the control plane"
+    echo "    token  - Show the dashboard credentials token"
+    echo ""
+}
+
+
+# Quick check to see if Helm is installed, as
+# It is used for service deployments on the Control Plane
+function helm_sanity_check {
+    # Is Helm installed?
+    echo "🔍 Check for Helm tool on the machine"
+    if ! command -v helm >/dev/null 2>&1; then
+        echo "❌ Helm not found; install helm before continuing"
+        exit 1
+    fi
+}
+
+
+# Is the dashboard service up and running?
+function dashboard_sanity_check {
+    echo "🔍 Checking if Service '${SERVICE}' exists in namespace '${NAMESPACE}'..."
+    if ! kubectl get svc -n "${NAMESPACE}" "${SERVICE}" >/dev/null 2>&1; then
+        echo "❌ Service ${SERVICE} NOT found in namespace ${NAMESPACE}"
+        exit 1
+    else
+        echo "✅ Service ${SERVICE} found in namespace ${NAMESPACE}"
+    fi
+}
+
+
+# Configure the control plane to be the home of the Kubernetes Dashboard
+# This is just fine for a small deployment for development, however proper
+# HA configuration should be used for a larger cluster
```

Comment:
This bit of magic is removing the restriction, allowing the operation of pods on the control plane.
It's relaxed ONLY for the "kube dashboard" namespace, so not open to anything possible.